src/flavors/eu/components/ecl-footers-eu/ecl-footers-eu.config.js

```diff
@@ -10,8 +10,9 @@ module.exports = {
         wrapper_class: 'ecl-footer__site-identity',
         menus: [
           {
+            extra_classes: 'ecl-col-sm',
             title: {
-              classes: 'ecl-h4',
+              classes: 'ecl-h4 ecl-footer__link--side-identity',
```

Comment:
I'd suggest "side" to be "site" to be move consistent with `.ecl-footer__site-identity`, if there is no other specific reasons to be "side" of course.